app/src/main/java/com/calendar/MainActivity.java

```diff
@@ -146,6 +147,10 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
                         (datePicker1, year, month, day) -> {
                             CalendarUtils.selectedDate = LocalDate.of(year, month + 1, day);
                             onResume();
+                            Fragment calendarFragment = fragmentManager.findFragmentByTag("calendarFragment");
+                            if(calendarFragment != null && calendarFragment.isAdded()) {
```

Comment:
I would propose to move this condition from if to new method.

Reply:
Moved to a method.